bootstrap/values-bootstrap.yaml

```diff
@@ -59,6 +59,14 @@ argocd-operator:
     prometheus:
       version: prometheusoperator.0.32.0
 
+  # argocd rbac only in listed namespaces
+  namespaceRoleBinding:
+    enabled: true
+    namespaces:
+    - name: *ci_cd
+    - name: *dev
+    - name: *test
```

Comment:
Our ArgoCD app is probs going to only manage things in `labs-test` and `labs-staging` right? Maybe those could be better defaults

Reply:
it needs all the projects that the argocd service account will need .. i will add staging as well.

it needs *ci_cd for sure - that where the UJ apps live
it needs *dev - else you wont be able to interact from argo to here for YOUR projects apps so seems sane to include